app/BUILD.bazel

```diff
@@ -43,6 +43,8 @@ Place your file here if:
 LISTENERS = [
     "src/main/java/org/oppia/app/administratorcontrols/RouteToAppVersionListener.kt",
     "src/main/java/org/oppia/app/administratorcontrols/RouteToProfileListListener.kt",
+    "src/main/java/org/oppia/app/administratorcontrols/LoadAppVersionListener.kt",
+    "src/main/java/org/oppia/app/administratorcontrols/LoadProfileListListener.kt",
```

Comment:
Please keep them in alphabetical order.

Reply:
If build passes, you can resolve this comment on your own.

Reply:
It is still failing, I don't know why.  
I have checked the details of the error but I can't figure out what the problem is.  
```
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
##[error]Process completed with exit code 1.
```

Reply:
Worked after adding `artifact("androidx.drawerlayout:drawerlayout:1.1.0"),` to BUILD.bazel as you suggested